src/plugins/sailfishoswizards/common.cpp

```diff
@@ -59,6 +59,25 @@ QString processTemplate(const QString &templateName, const QHash<QString, QStrin
     return Utils::TemplateEngine::processText(&expander, templateText.readAll(), errorMessage);
 }
 
+/*!
+ * \brief Returns the path to .pro file from dir .
+ */
+QString getProFilePath(const QString &applicationDir)
+{
```

Comment:
This seems both a bit too complicated (I don't think it is necessary to check that a file exists if it is in the directory entry list..) and too simple (using the pro file with the same name as the directory should probably have higher priority than any other pro files).